cointop/config.go

```diff
@@ -48,6 +48,7 @@ type ConfigFileConfig struct {
 	Colorscheme       interface{}            `toml:"colorscheme"`
 	RefreshRate       interface{}            `toml:"refresh_rate"`
 	CacheDir          interface{}            `toml:"cache_dir"`
+	ScaleNumbers      interface{}            `toml:"scale_numbers"`
```

Comment:
I think calling it `compact_notation` instead of `scale_numbers` could make more sense. For example, that's the terminology used in the browser Intl formatter:

![DeepinScreenshot_select-area_20211003220942](https://user-images.githubusercontent.com/168240/135797507-a8286fd8-36ee-4d0a-a7e0-d1f2d2c09a1d.png)


If the config option is global then maybe all values should be in the compact notation (ie tables, chart, marketbar)? If it's only for the tables, then moving the config option to the `table` config would make sense (similar to  `keep_row_focus_on_sort` option that affects all tables)


![DeepinScreenshot_select-area_20211003221040](https://user-images.githubusercontent.com/168240/135797512-693718cd-ad58-480c-9ae5-c0df5475c354.png)

Reply:
Renamed...

Reply:
OK, made separate configuration for coin table, favorites table, portfolio table and market bar...

THEN figured out there's no compactable fields on portfolio. Ha!